charts/catalog/values.yaml

```diff
@@ -44,6 +44,9 @@ apiserver:
       globalNamespace: servicecatalog
   # Log level; valid values are in the range 0 - 10
   verbosity: 10
+  # disable authentication so that people don't have to bother setting up certificates
+  # properly just to get the walkthrough working
+  enableAuth: false
```

Comment:
Two nits here. To be consistent with the rest of the comments in the `values.yaml` file, we should upcase the leading letter of this comment. Also, for consistency, we should probably be documenting the function of the switch instead of the default _value_ of the switch. i.e. Instead of saying "disable authentication ..." (which documents the default value), say instead "Enable or disable authentication ... Disabling can be useful for kicking the tires in a local cluster, but is not suitable for production."